"rust2go/asmcall" imported and not used

Seems like asmcall is imported even if not used anywhere, breaking the auto-gen build.

```
# r2gexample/src-tauri/go
  ./gen.go:31:2: "github.com/ihciah/rust2go/asmcall" imported and not used
```

Repro & workaround shown below:

```rust
#[rust2go::r2g]
pub trait Runner {
    fn start();
    #[cgo_callback]
    fn next() -> Output;
    fn _unused() -> Output; // Comment out to trigger the generation error
    fn send(req: String);
    fn close();
}
```

```rust
use rust2go::RegenArgs;

fn main() {
    rust2go::Builder::new()
        // .with_regen(src, dst)
        .with_go_src("./go")
        .with_regen_arg(RegenArgs {
            src: "./src/agent.rs".into(),
            dst: "./go/gen.go".into(),
            ..Default::default()
        })
        .build();
}
```

--- a/rust2go-cli/src/lib.rs
+++ b/rust2go-cli/src/lib.rs
@@ -85,7 +85,8 @@ pub fn generate(args: &Args) {
     let use_cgocall =
         r2g_any!(|f| f.mem_call_id().is_none() && f.cgo_callback()) || g2r_any!(|f| f.cgo_call());
     let use_asmcall =
-        r2g_any!(|f| f.mem_call_id().is_none() && !f.cgo_callback()) || g2r_any!(|f| !f.cgo_call());
+        r2g_any!(|f| f.mem_call_id().is_none() && !f.cgo_callback() && f.ret().is_some())
+            || g2r_any!(|f| !f.cgo_call());
     if use_shm {
         importc.push_str(RawRsFile::go_shm_include());
     }

